packages/opencensus-exporter-jaeger/src/jaeger-driver/index.ts

```diff
@@ -48,16 +48,39 @@ export type SenderCallback = (numSpans: number, err?: string) => void;
  * @param span
  */
 export function spanToThrift(span: Span) {
-  let spanTags = [];
+  const tags = [];
   if (span.attributes) {
-    const tags = [];
     Object.keys(span.attributes).forEach(key => {
       tags.push({'key': key, 'value': span.attributes[key]});
     });
-    spanTags = ThriftUtils.getThriftTags(tags);
   }
 
-  const spanLogs = [];
+  if (span.messageEvents) {
+    span.messageEvents.forEach(msg => {
+      tags.push({'key': 'message.id', 'value': msg.id});
```

Comment:
Is it OK for multiple values to exist under the same key here? Here and for annotations as well.

Reply:
Since these are getting translated to individual log events with tags, rather than just tags, it should be fine. I'm less sure what happens if you have a log event without a timestamp (which would be the case for message events).
 I don't really have a good grasp of messageEvents in general, but dropping them would seem like the wrong thing to do.
  I've not had a chance to actually run this yet, shouldn't have PR'd, sorry.